core/primitives/res/storage_usage_delta.csv

```diff
@@ -0,0 +1,3112 @@
+viktorliu.near,1
+viktorskf.near,1
+vikusss89.near,1
+vilija.near,2
+vincent.near,1
+vinilpc.near,1
+vinmar.near,3
+vip.near,1
+vitalik.near,1
+vitalik_buterin.near,1
+vitaliy.near,1
+vitaly.near,1
+vitas86.near,1
+vitkov.near,1
+vitkov2.near,1
+vivi1333.near,1
+vk1118.near,1
+vlad.near,1
+vladimirg.near,1
+vladis.near,1
+vladsmd.near,2
+voland04.near,1
+vote4pedro.near,1
+vovusik.near,1
+vox.near,1
+vy_vy_22.near,1
+vytas.near,1
+wang16.near,2
+wangchao86.near,1
+wanglei0095.near,1
+wangsta.near,1
+wanseob.near,1
+warisyaqubi.near,2
+warlock.near,1
+warmcapital.near,1
+was.near,3
+waters.near,1
+wearesatoshi.near,1
+weariedatom.near,3
+web3.near,1
+webmel.near,2
+weed.near,1
+weiliangfung.near,2
+wenjielu.near,2
+wenxianjiang.near,1
+wetez.near,1
+whatever.near,1
+whiletrue.near,2
+white_house.near,3
+whoismrbru.near,1
+whoismrskill.near,1
+wildalert.near,1
+willymooth.near,1
+wimscot123.near,2
+win.near,1
+winnerclubs.near,1
+winnieyow.near,1
+wistar.near,1
+wladka666.near,3
+wodikasha.near,1
+wojak.near,1
+wolf34.near,1
+wompwomp.near,1
+wonbo.near,1
+wonderreturn.near,1
+woodwam.near,1
+wookiewarrior.near,2
+wow.near,1
+wowanjd.near,1
+wr.near,1
+wtldmrs.near,2
+wu365.near,1
+wu40.near,1
+wudi.near,2
+wuffle.near,1
+wurstmensch.near,1
+ww.near,1
+wx.near,2
+x.near,1
+xafobia.near,2
+xander.near,1
+xanyar.near,2
+xbadmax.near,2
+xbleidx11.near,1
+xedap.near,1
+xianghong0912.near,1
+xiaobo.near,1
+xiaojun.near,1
+xikunying.near,1
+xoreth.near,8
+xpaco.near,2
+xpromt.near,1
+xrahul.near,1
+xtreme11.near,1
+xtreme12.near,1
+xuyogi.near,1
+xvideos469.near,2
+xx.near,1
+xxx.near,1
+xy.near,2
+xyb.near,2
+y0b.near,1
+yajirushiworks2020.near,2
+yakultza.near,2
+yam.near,1
+yams.near,4
+yan.near,5
+yangchj.near,1
+yangshen.near,1
+yangweizi911.near,1
+yanmingguo.near,1
+yanok1985.near,2
+yaomengmeng1986.near,1
+yarco.near,1
+yashman.near,1
+yasser.near,2
+ybob.near,2
+yc.near,2
+ycl.near,2
+yderdiki.near,1
+yearn.near,1
+yellowboy.near,1
+yessin.near,2
+yfi.near,1
+yfledger.near,1
+yicrson.near,1
+yin123y.near,3
+yinqa.near,1
+ykhan60.near,1
+ykvph.near,1
+yl.near,1
+yoda.near,3
+yoda_uk.near,2
+yodake.near,2
+yolo_surf.near,1
+yongming.near,5
+yoonho00.near,1
+york1ng99.near,1
+youtube.near,2
+yrl2000.near,1
+ysxlin01.near,2
+yt8686jh.near,1
+yuchialiang.near,2
+yuggalen.near,1
+yujinbo.near,1
+yulian.near,1
+yuridiablo.near,2
+yuristturist.near,1
+yuriy.near,1
+yury.near,1
+yuta.near,1
+yuther.near,1
+yy.near,1
+yylai.near,2
+z.near,1
+zachzwei.near,1
+zaebumba.near,1
+zamzam1.near,1
+zamzam2.near,1
+zano66.near,1
+zaremba.near,1
+zazoley.near,2
+zealot.near,1
+zeecity.near,1
+zenithli.near,1
+zeromax.near,1
+zeroone.near,1
+zeus7912.near,4
+zeusone.near,1
+zeze.near,2
+zhartur89.near,1
+zhelserhii09.near,1
+zhuhao529440.near,1
+zhuoyue.near,1
+ziaaddils.near,1
+zilj.near,1
+zinur.near,1
+zitko.near,2
+ziyou.near,1
+zjouraolian.near,1
+zkp.near,2
+zoek.near,1
+zorro.near,1
+zpool.near,4
+zyzygy.near,1
+zzangpk.near,1
+zzf.near,1
+zztom.near,1
```

Comment:
I'm not totally understanding the background here and from this PR my guess is some user's storage usage is wrong and adjust by this file. Will there be more accounts have incorrect storage since this file is made?

Reply:
The bug that led to this discrepancy was fixed

Reply:
The original bug was already fixed, so delta of storage usage should not change